data/repos.yml

```diff
@@ -95,6 +98,20 @@
   type: Utilities
   team: "#govuk-platform-reliability-team"
 
+- repo_name: cdn_users_step_function
+  # TODO: archive. 2 commits, 2 years ago, doesn't appear to be used.
+
+- repo_name: ckan
+  # It's 211 commits behind `ckan:master`, but presumably we're using a fork to prevent
+  # third parties from being able to tamper with it. We reference the fork here:
+  # https://github.com/alphagov/ckanext-datagovuk/blob/4ae4e0eb03c88b4526672527490a6546e95c4d78/docker/ckan/Dockerfile#L69-L72
+  # ...which references this commit within the fork:
+  # https://github.com/alphagov/ckan/commit/0d714b258668ee78a0b19182c53b34689629df37
+  type: data.gov.uk apps
+  team: "#govuk-datagovuk"
+  sentry_url: false
+  dashboard_url: false
```

Comment:
@kentsanggds would you mind double-checking my understanding here? This `ckan` fork _is_ actively used and should be in the dev docs I think?

Reply:
yes, unfortunately we are still using a fork of it on the old puppet stack, the new platform will be using the upstream CKAN so then we can close the fork down.